src/pages/Market/MarketTable/index.tsx

```diff
@@ -0,0 +1,155 @@
+/** @jsxImportSource @emotion/react */
+import React, { useContext, useMemo } from 'react';
+import { Typography } from '@mui/material';
+import { AuthContext } from 'context/AuthContext';
+import { Table, Token, ITableProps, LayeredValues } from 'components';
+import { useTranslation } from 'translation';
+import { Asset, TokenId } from 'types';
+import { useUserMarketInfo } from 'clients/api';
+import {
+  formatCoinsToReadableValue,
+  formatCentsToReadableValue,
+  formatToReadablePercentage,
+} from 'utilities/common';
+import { useStyles as useSharedStyles } from '../styles';
+import { useStyles as useLocalStyles } from './styles';
+
+export interface IMarketTableProps extends Pick<ITableProps, 'rowOnClick'> {
+  assets: Asset[];
+}
+
+export const MarketTableUi: React.FC<IMarketTableProps> = ({ assets, rowOnClick }) => {
+  const { t } = useTranslation();
+  const sharedStyles = useSharedStyles();
+  const localStyles = useLocalStyles();
+  const styles = { ...sharedStyles, ...localStyles };
+
+  const columns = useMemo(
+    () => [
+      { key: 'asset', label: t('market.columns.asset'), orderable: false },
+      { key: 'totalSupply', label: t('market.columns.totalSupply'), orderable: true },
+      { key: 'supplyApy', label: t('market.columns.supplyApy'), orderable: true },
+      { key: 'totalBorrows', label: t('market.columns.totalBorrow'), orderable: true },
+      { key: 'borrowApy', label: t('market.columns.borrowApy'), orderable: true },
+      { key: 'liquidity', label: t('market.columns.liquidity'), orderable: true },
+      { key: 'price', label: t('market.columns.price'), orderable: true },
+    ],
+    [],
+  );
+
+  const cardColumns = useMemo(() => {
+    const newColumns = [...columns];
+    const [liquidityCol] = newColumns.splice(5, 1);
+    newColumns.splice(3, 0, liquidityCol);
+    return newColumns;
+  }, [columns]);
+
+  // Format assets to rows
+  const rows: ITableProps['data'] = assets.map(asset => [
+    {
+      key: 'asset',
+      render: () => <Token symbol={asset.symbol as TokenId} />,
+      value: asset.id,
+    },
+    {
+      key: 'totalSupply',
+      render: () => (
+        <LayeredValues
+          topValue={formatCentsToReadableValue({
+            value: asset.treasuryTotalSupplyUsdCents,
+            shorthand: true,
+          })}
+          bottomValue={formatCoinsToReadableValue({
+            value: asset.treasuryTotalSupplyUsdCents.div(asset.tokenPrice.times(100)),
+            tokenId: asset.id as TokenId,
+            shorthand: true,
+          })}
+        />
+      ),
+      value: asset.treasuryTotalSupply.toFixed(),
+    },
+    {
+      key: 'supplyApy',
+      render: () => (
+        <LayeredValues
+          topValue={formatToReadablePercentage(asset.supplyApy)}
+          bottomValue={formatToReadablePercentage(asset.xvsSupplyApy)}
+        />
+      ),
+      value: asset.borrowBalance.toFixed(),
```

Comment:
I checked on the current implementation and the top value needs to include the XVS supply APY
```suggestion
        <LayeredValues
          topValue={formatToReadablePercentage(asset.supplyApy.plus(asset.xvsSupplyApy))}
          bottomValue={formatToReadablePercentage(asset.xvsSupplyApy)}
        />
      ),
      value: asset.supplyApy.plus(asset.xvsSupplyApy).toFixed(),
```